app/controllers/redirection_controller.rb

```diff
@@ -7,6 +7,16 @@ def redirect_covid
     redirect_to(finder_path(params[:slug], params: covid_topic_and_other_params))
   end
 
+  def redirect_latest
+    redirect_params = {
+      order: "updated-newest",
+    }
+    redirect_params[:organisations] = params[:departments] if params[:departments]
+    redirect_params[:topical_events] = params[:topical_events] if params[:topical_events]
+    redirect_params[:world_locations] = params[:world_locations] if params[:world_locations]
```

Comment:
Just a thought, I'm wondering here if there's a way of only adjusting the departments param?

Was playing around with something like 

`params.transform_keys { |key| key == :departments ? :organisations : key }`

but getting non permitted param errors, so maybe need to filter for only permitted ones.

Basically, the motivation being that we aren't actually doing anything to the topical events or world location params, so feels noisy to being doing this. Also do we have a guarantee that these are the only params we'll want to pass through?